libraries/botframework-connector/src/auth/botFrameworkClientImpl.ts

```diff
@@ -26,9 +26,8 @@ const botFrameworkClientFetchImpl: typeof fetch = async (input, init) => {
     } as Response;
 };
 
-// Internal
-/**
- * Implementation of BotFrameworkClient.
+/**@internal
+ * Implementation of [BotFrameworkClient](xref:botframework-connector.BotFrameworkClient).
```

Comment:
This should be in its own separate line, see other cases in the project. same for the rest
```
/**
 * @internal
 * Implementation of [BotFrameworkClient](xref:botframework-connector.BotFrameworkClient).
```

Reply:
Also ensure the PR checks are passign successfully